lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -230,6 +230,20 @@ private void requireClaim(String name, Object value) {
         }
     }
 
+    private static boolean checkVarargNull(String[] args) {
+        if (args == null || args.length == 0) {
+            return true;
+        }
+        boolean isAllNull = true;
+        for (String arg: args) {
+            if (arg != null && arg.trim().length() != 0) {
```

Comment:
Let's not check for empty strings here, just null. That will be consistent with how we handle empty strings for other claims.